setup.py

```diff
@@ -151,6 +152,9 @@ def _parse_requirements_file(file_path):
     "scikit-learn",
     "seqeval",
 ]
+_sentence_transformers_integration_deps = [
+    "optimum-deepsparse @ git://github.com/neuralmagic/optimum-deepsparse.git"
+] + _torch_deps
```

Comment:
@dhuangnm will this build into a pypi wheel? Think it may not

Reply:
I think we'll probably need to build optimum-deepsparse wheel separately as we do for nm-transformers and nm-yolov5, then push it out to public pypi in order to install from it?

Reply:
if we want `pip install deepsparse[sentence-transformers]` to work from pypi this is probably an option we'll have to consider